src/configuration/vimrc.ts

```diff
@@ -42,33 +42,54 @@ class VimrcImpl {
    * Adds a remapping from .vimrc to the given configuration
    */
   private static addRemapToConfig(config: IConfiguration, remap: IVimrcKeyRemapping): void {
-    const remaps = (() => {
+    const mappings = (() => {
       switch (remap.keyRemappingType) {
+        case 'map':
+          return [
+            config.normalModeKeyBindings,
+            config.visualModeKeyBindings,
+            config.insertModeKeyBindings,
```

Comment:
According to [this](https://vim.fandom.com/wiki/Mapping_keys_in_Vim_-_Tutorial_(Part_1)) and a bit of testing, seems `map` doesn't create a mapping for insert/command line modes.

Reply:
Ah, thanks.  Good catch,

Fixed.

Reply:
Hey @jjoekoullas did you push your changes? I'm still seeing `config.insertModeKeyBindings`

Reply:
Ah, actually I just missed it.  Fixed now.